draft-ietf-ppm-dap.md

```diff
@@ -2477,6 +2476,17 @@ disagrees). See
 [#22](https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/issues/22) for
 discussion.
 
+## Verification key requirements {#verification-key}
+
+The verification key for a task SHOULD be chosen before any reports are generated.
+It SHOULD be fixed for the lifetime of the task and cannot be rotated. One way
+to ensure this is to include the verification key in a derivation of the task ID.
+
+To ensure that the security proofs for Prio3 hold, the verification key MUST be
```

Comment:
I think it's worth citing Prio3 as an example, but in the interest of being generic, I think we should add a little more context here.

```suggestion
This consideration comes from current security analysis for existing VDAFs. For example,
to ensure that the security proofs for Prio3 hold, the verification key MUST be
```